src/NSwag.SwaggerGeneration.AspNetCore/AspNetCoreToSwaggerGenerator.cs

```diff
@@ -187,40 +201,6 @@ public static JsonSerializerSettings GetJsonSerializerSettings(IServiceProvider
             List<Tuple<SwaggerOperationDescription, ApiDescription, MethodInfo, IEnumerable<string>, IEnumerable<string>>> operations,
```

Comment:
I think we can remove the last two parameters of the tuple, right?

Reply:
Well, I think we should leave them because we need consume/produce values of the operation to check if they exists in a global section. And if they exist, we haven't to create the same sections in the operation scope.

Reply:
Ah ok... reviewing on my phone and not in an ide 🙂